jetty9/src/main/docker/setup-env-ext.bash

```diff
@@ -34,4 +34,9 @@ if ! type "$1" &>/dev/null; then
   export JAVA_OPTS="$JAVA_OPTS $JETTY_ARGS"
 fi
 
+# If we are deployed on a GAE platform, enable the gae module
+if [ -n "$GAE_INSTANCE" -a ! -e "$JETTY_BASE/start.d/gae.ini" ]; then
+  echo "--module=gae" > $JETTY_BASE/start.d/gae.ini
```

Comment:
Is Jetty scanning `$JETTY_BASE/start.d/*.ini` for configuration / adding modules at startup?
Is that better than modifying `jetty.commands`? Or, could we have used `gcp.ini` instead of `jetty.commands`?